### PR TITLE
Keymodel

### PR DESCRIPTION
**Pull request by LaneScobie11**
Keywords.py takes in a faculty object and uses the faculty_id to get all of that profs documents on elastic search then returns a list of the keyword objects. The function UpdateKeywordsFromGenerator receives this list of keyword objects and does the actual updating/posting to elasticSearch. 

-just run workflow.

### CODE REVIEW

File: web/bfex/app.py [hunk 1]
```diff
@@ -27,8 +30,18 @@ def create_app():
 
     initialize_models()
 
+<<<<<<< HEAD
     app.register_blueprint(faculty_bp)
     app.register_blueprint(search_bp)
+    #register_approach(GenericApproach, 0)
+    #register_approach(RakeApproach, 1)
+=======
```
Comment (GarryBullock) [inline web/bfex/app.py]: Why are you checking in merge conflicts?

File: web/bfex/components/data_pipeline/tasks/keywords.py [hunk 2]
```diff
@@ -4,68 +4,64 @@
...
-class UpdateKeywordsFromScrape(Task):
+class GetKeywordsFromScrape(Task):
     """
     Updates Keywords of a Faculty Members data in elastic.
     """
     def __init__(self):
         self.task_name = "Update Keywords From Scrape"
 
-    def is_requirement_satisfied(self, data):
+    def is_requirement_satisfied(self, faculty):
```
Comment (GarryBullock) [inline web/bfex/components/data_pipeline/tasks/keywords.py]: Every function needs a docstring, even if its a one liner.

File: web/bfex/components/data_pipeline/tasks/keywords.py [hunk 3]
```diff
@@ -4,68 +4,64 @@
...
-        
-            faculty = search_results[0]
-            if faculty.text != None:
-                
-                rake = RakeApproach()
-                rake_keyword = rake.generate_keywords(faculty.text)
-                faculty.rake_keywords = rake_keyword
+        key_objects= []
+        faculty_id = faculty.faculty_id
+        time.sleep(1)
```
Comment (GarryBullock) [inline web/bfex/components/data_pipeline/tasks/keywords.py]: Is this to circumvent the data not saving? We might be better off adding a delay between tasks in Workflow

Comment (LaneScobie11) [reply web/bfex/components/data_pipeline/tasks/keywords.py]: yes, this is to give it enough to post to elasticsearch before indexing the data, a delay between tasks in workflow would probably be better.

**Pull request merged**